ex9/js/main.js

```diff
@@ -0,0 +1,38 @@
+requirejs(['ajaxGet','load','filter', 'sort'] , function(ajaxGet, load, filter, sort) {
+  $(function() {
+    var phoneBookData = {};
+    var render = function(objList) {
+      $('li').remove();
+      $("#phoneBookTemplate").tmpl( objList )
+      .appendTo( "#phoneBook" );
+
+    // Remove li element from list
+    $("li").on("click",function(elementID) {
+      console.log('testing',elementID);
+      $(this).remove();
+    });
+
+
+    };
+
+    load(function() {
+      ajaxGet('json/data.json', function(phoneBookDataA) {
+        phoneBookData = phoneBookDataA;
+        render( phoneBookData );
+      })
+    });
+
+    filter(function() {
+      var filterList = _.filter(phoneBookData, function(name) {
+        if (name.name.indexOf($( '#filter' ).val()) > -1) return true;
+      });
+      render( _.sortBy(filterList, 'name'));
+    });
+
+    sort(function() {
```

Comment:
Since the sortAsc and sortDesc are similar, I'd make them use the same callback with some param passed (for example: `sort({asc: true});` or `sort({asc: false});`.
This makes even more sense when there's more info you wish to convey to the callback, like in filter (jQuery use objects for almost everything, think `$.ajax`).